src/rgw/driver/rados/rgw_zone.h

```diff
@@ -122,6 +122,7 @@ struct RGWZoneParams : RGWSystemMetaObj {
   std::string realm_id;
 
   JSONFormattable tier_config;
+  JSONFormattable sal_config;
```

Comment:
@dang I ended up choosing `sal_config` for the name of the json that will be part of the zone's info. Do you have another suggestion for the name? `driver_config`?

Reply:
I think sal_config is fine.  It's more than drivers (filters too), so SAL is probably appropriate.